documentation/doxygen/makeimage.py

```diff
@@ -12,6 +13,7 @@ def makeimage(MacroName, ImageName, OutDir, cp, py, batch):
     if batch:
         ROOT.gROOT.SetBatch(1)
 
+    sys.argv = [MacroName]
```

Comment:
this should be under `if py`

Reply:
ok. I will rerun de doc generation with that change

Reply:
done